<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addbook</class>
 <widget class="QDialog" name="Addbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color:rgb(111, 76, 19);
font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="title">
      <string>Add new book</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_title</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_title">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Author</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_author</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_author">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Quantity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_quantity">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Genre</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_quantity</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>Biography</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mystery</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>History</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Science</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Horror</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Travel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Business</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Children</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cooking</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_location">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Location</string>
            </property>
            <property name="buddy">
             <cstring>comboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_location">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
background: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
